test/integration-test/src/tests/load.rs

```diff
@@ -165,4 +198,43 @@ fn pin_lifecycle() {
 
     // program should be unloaded
     assert_loaded!("pass", false);
+
+    //delete veth interface pait
+    delete_veth_pair(veth_name).unwrap();
+}
+
+fn create_veth_pair(veth_name: &str) -> Result<(), rtnetlink::Error> {
+    let (_, handle, _) = new_connection().unwrap();
+    block_on(async {
```

Comment:
Just an idea, which could be done in another PR, but you could do it even in this one - what about making all tests async? We already use tokio by default in userspace crates in aya-template.

Reply:
I could do that if we deem that it makes sense. I am new to the project so I didn't want to fundamentally change what was happening.